MNE/disp3D/3DObjects/digitizer/digitizersettreeitem.cpp

```diff
@@ -0,0 +1,290 @@
+        itemList << digitizerItem;
+        itemList << new QStandardItem(digitizerItem->toolTip());
+        this->appendRow(itemList);
+        itemList.clear();
+
+    }
+    if (!tHpi.empty()){
+
+        //Create a HPI digitizer item
+        DigitizerTreeItem* digitizerItem = new DigitizerTreeItem(Data3DTreeModelItemTypes::DigitizerItem,"HPI");
+        state = digitizerItem->addData(tHpi, parent);
+        itemList << digitizerItem;
+        itemList << new QStandardItem(digitizerItem->toolTip());
+        this->appendRow(itemList);
+        itemList.clear();
+    }
+    if (!tEeg.empty()){
+
+        //Create a EEG digitizer item
+        DigitizerTreeItem* digitizerItem = new DigitizerTreeItem(Data3DTreeModelItemTypes::DigitizerItem,"EEG/ECG");
+        state = digitizerItem->addData(tEeg, parent);
+        itemList << digitizerItem;
+        itemList << new QStandardItem(digitizerItem->toolTip());
+        this->appendRow(itemList);
+        itemList.clear();
+    }
+    if (!tExtra.empty()){
+
+        //Create a extra digitizer item
+        DigitizerTreeItem* digitizerItem = new DigitizerTreeItem(Data3DTreeModelItemTypes::DigitizerItem,"Extra");
+        state = digitizerItem->addData(tExtra, parent);
+        itemList << digitizerItem;
+        itemList << new QStandardItem(digitizerItem->toolTip());
+        this->appendRow(itemList);
+        itemList.clear();
+
+    }
+
+//    this->appendRow(itemList);
+    QList<QStandardItem*> test=this->findChildren("EEG/ECG");
+
+    return state;
+}
+
+
+//*************************************************************************************************************
+
+void DigitizerSetTreeItem::setVisible(bool state)
+{
+//    m_pRenderable3DEntity->setParent(state ? m_pParentEntity : Q_NULLPTR);
+
+    for(int i = 0; i<m_lChildren.size(); i++) {
+        m_lChildren.at(i)->setParent(state ? m_pRenderable3DEntity : Q_NULLPTR);
+    }
+}
+
+
+//*************************************************************************************************************
+
+void DigitizerSetTreeItem::onSurfaceColorChanged(const QColor& color)
+{
+    QVariant data;
+    QByteArray arrayNewVertColor = createVertColor(this->data(Data3DTreeModelItemRoles::SurfaceVert).value<MatrixX3f>(), color);
+
+    data.setValue(arrayNewVertColor);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceCurrentColorVert);
+}
+
+
+//*************************************************************************************************************
+
+void DigitizerSetTreeItem::onCheckStateChanged(const Qt::CheckState& checkState)
+{
+    for(int i = 0; i<this->rowCount(); i++) {
+        if(this->child(i)->isCheckable()) {
+            this->child(i)->setCheckState(checkState);
+        }
+    }
+}
+
+
+//*************************************************************************************************************
+
+QByteArray DigitizerSetTreeItem::createVertColor(const MatrixXf& vertices, const QColor& color) const
```

Comment:
Do you need this function?  If not, please remove from ehre and header file as well